app/src/main/java/org/dhis2/usescases/eventsWithoutRegistration/eventCapture/EventCapturePresenterImpl.java

```diff
@@ -602,7 +602,9 @@ public void displayMessage(String message) {
     @SuppressLint("CheckResult")
     @Override
     public void saveImage(String uuid, String filePath) {
-        valueStore.save(uuid, filePath).blockingFirst();
+        if (uuid != null) {
```

Comment:
What would happen if uid is null, image wouldn't be saved? is there any feedback to the user?

Reply:
This only prevents the crash, the uid should never be null, but I was not able to reproduce it. If uid is null nothing would happen.